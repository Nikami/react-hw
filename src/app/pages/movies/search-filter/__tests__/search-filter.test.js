import React from 'react';
import { shallow } from 'enzyme';
import SearchFilter from '../index';

describe('SearchFilter', () => {
  it('matches snapshot', () => {
    const defaultProps = 10;
    const wrapper = shallow(<SearchFilter moviesCount={defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
