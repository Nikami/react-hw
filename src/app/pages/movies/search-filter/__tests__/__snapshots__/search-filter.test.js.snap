// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFilter matches snapshot 1`] = `
<WithStyles(FilterContainer)
  contentClasses="search-filter"
>
  <div
    className="search-filter__text font-bold"
  >
    10
     movies found
  </div>
  <div
    className="search-filter__toolbar"
  >
    <div
      className="search-filter__text text-right font-bold"
    >
      Sort by
    </div>
    <WithStyles(Button)
      color="secondary"
      size="small"
    >
      release date
    </WithStyles(Button)>
    <WithStyles(Button)
      color="primary"
      size="small"
    >
      rating
    </WithStyles(Button)>
  </div>
</WithStyles(FilterContainer)>
`;
